apps/test/netsim/NetSimTable.js

```diff
@@ -186,7 +188,7 @@ describe("NetSimTable", function () {
     assertEqual(apiTable.log(), '');
 
     // Until poll interval has passed.
-    netsimTable.lastFullUpdateTime_ = Date.now() - (netsimTable.pollingInterval_ + 1);
+    netsimTable.lastRefreshTime_ = Date.now() - (netsimTable.pollingInterval_ + 1);
```

Comment:
Not that it really matters much, but is the "+ 1" needed?  e.g. if the polling interval is 1 the current time is 10, the last refresh was at 9 not 8.